_drafts/_heat/using-ucd-ucdp.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Using IBM UrbanCode Deploy with IBM Blue Box
+featured: no
+weight: 10
+tags: [heat, UCD]
+author: Ying Tang, Pan Xia Zou
+dateAdded: June 7, 2016
+---
+
+IBM UrbanCode Deploy with Patterns (UCDP) is an environment management and deployment solution that you can use to deploy applications to multiple clouds, and manage their lifecycles. 
```

Comment:
@nirajdp76 Hi Niraj, do you think we can talk to UCD/UCDP guys and get their attention to describe or refer to Blue Box usage in their knowledge center? Also inviting @panxia6679 to comment.